### PR TITLE
media/dataSource : add format type enum to sync pcmformat in tinyalsa

### PR DESCRIPTION
**Pull request by Taejun-Kwon**

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [e688a7e4f304c4f8fa0e7893fd602ac19cc4f129] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [e688a7e4f304c4f8fa0e7893fd602ac19cc4f129] - Code Rule Check Result:
framework/include/media/DataSource.h:92: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/DataSource.h:92: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)

File: apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_fileoutputdatasource.cpp [hunk 1]
```diff
@@ -48,11 +48,11 @@ TEST_F(FileOutputDataSourceTest, setMemberVariablePositive)
 {
 	unsigned short compare_channels = 3;
 	unsigned int compare_sampleRate = 32000;
-	int compare_pcmFormat = 1;
+	int compare_pcmFormat = media::AUDIO_FORMAT_TYPE_S8;
 
 	dataSource->setChannels(compare_channels);
 	dataSource->setSampleRate(compare_sampleRate);
-	dataSource->setPcmFormat(compare_pcmFormat);
+	dataSource->setPcmFormat(media::AUDIO_FORMAT_TYPE_S8);
```
Comment (chanijjani) [inline apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_fileoutputdatasource.cpp]: Please check whether this needs to be "media::AUDIO_FORMAT_TYPE_S16_LE" or not.

Comment (jarvisjang) [reply apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_fileoutputdatasource.cpp]: In my opinion, this function checks only that the "set value" is same as the "get value". So no problems.

**Comment by jsdosa** [comment]
@Taejun-Kwon tj, please update setPcmFormat function.
setPcmFormat(int pcmFormat) --> setPcmForamt(AudioFormatType);
return type for getPcmFormat().

**Comment by seinfra** [comment]
Target : [0029bf1107f336eb37c6c281c0e2eac2b2416918] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [0029bf1107f336eb37c6c281c0e2eac2b2416918] - Code Rule Check Result:
framework/include/media/DataSource.h:92: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/DataSource.h:92: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)

**Pull request merged**